wow.go

```diff
@@ -108,3 +115,8 @@ func (w *Wow) PersistWith(s spin.Spinner, text string) {
 		fmt.Fprint(w.out, txt)
 	}
 }
+
+// ForceOutput forces all output even if not not outputting directly to a terminal
+func ForceOutput(w *Wow) {
+	w.isTerminal = true
```

Comment:
Seeing we only update values of single property. We can as well make `isTerminal` public. That way the user can explicitly  change it or use your option implementation.